pkg/service/rtcservice.go

```diff
@@ -373,7 +374,8 @@ func (s *RTCService) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 				case *livekit.SignalResponse_Update:
 					pLogger.Debugw("sending participant update", "participantUpdate", m)
 				case *livekit.SignalResponse_RoomMoved:
-					loggerResolver("room", m.RoomMoved.GetRoom(), "pID", m.RoomMoved.GetParticipant().GetSid())
+					loggerResolver.Reset()
+					loggerResolver.Resolve("room", m.RoomMoved.GetRoom(), "pID", m.RoomMoved.GetParticipant().GetSid())
```

Comment:
did we want to wait to resolve this until later so the intervening logs get the new room id?

Reply:
oh... i see... my bad, not paying close enough attention. can we repeat the path on controller that we do to resolve these initially - wait for the room update with the id then set everything?

Reply:
Will look at it. Do all room updates have resolved ID only? So, till room resolves, the updates have empty ID?

Reply:
yes, until the room resolves there may be updates with an empty id

Reply:
sweet, I will then hold till roomID is available to resolve.

And we can do the same thing for move also.